style.css

```diff
@@ -0,0 +1,439 @@
+}
+
+.link__white {
+  color: #fff;
+  text-align: left;
+}
+
+.arrow {
+  padding-left: 7px;
+}
+
+/* catalog */
+
+.catalog {
+  margin-bottom: 52px;
+}
+
+.catalog__title {
+  font-size: 22px;
+  font-weight: bold;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  margin-bottom: 15px;
+}
+
+.catalog__items {
+  display: flex;
+  justify-content: space-between;
+}
+
+.catalog__item {
+  display: block;
+  position: relative;
+  width: 208px;
+  height: 261px;
+}
+
+.catalog__item:not(:last-child)::after {
+  content: "";
+  opacity: 0.2;
+  background: #000;
+  height: 101px;
+  position: absolute;
+  top: 36px;
+  right: -15px;
+  width: 1px;
+}
+
+.catalog-item__image {
+  width: 170px;
+  height: 173px;
+  margin-bottom: 24px;
+  padding-left: 21px;
+}
+
+.catalog-item__title {
+  font-size: 12px;
+  line-height: 11px;
+  text-align: left;
+  margin-bottom: 10px;
+  padding-left: 11px;
+}
+
+.catalog-item__description {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding-left: 11px;
+}
+
+.catalog-item__price {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 13px;
+  text-align: left;
+}
+
+.catalog-item__varities {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.catalog-item__grey {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #c4c4c4;
+}
+
+.catalog-item__black {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #0e1211;
+}
+
+.catalog-item__red {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #b62e3c;
+}
+
+.catalog-item__orange {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #fdb431;
+}
+
+.catalog-item__brown {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #e29d69;
+}
+
+.catalog-item__cherry {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #762e31;
+}
+
+.catalog-item__blue {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #257da9;
+}
+
+.active-item {
+  height: 9px;
+  width: 9px;
+}
+
+.gradient {
+  display: none;
+}
+
+.gradient-reverse {
+  display: none;
+}
+
+/* video-section */
+
+.video-section {
+  width: 100%;
+  height: 442px;
+  background-color: #000;
+  display: flex;
+  align-items: center;
+
+  /* padding-left: 30px; */
+  padding-bottom: 95px;
+  justify-content: space-around;
+}
+
+.video-section__image {
+  display: block;
+}
+
+.video-adaptive {
+  display: none;
+}
+
+.video-section__content {
+  display: block;
+}
+
+.video-content__h3 {
+  width: 258px;
+  font-size: 22px;
+  line-height: 27px;
+  margin-bottom: 12px;
+  color: #fff;
+  text-transform: uppercase;
+}
+
+.video-content__description {
+  width: 248px;
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 10px;
+  color: #fff;
+}
+
+/* stories-content */
+
+.stories-content { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)